harvester/records/formats/marc.py

```diff
@@ -148,11 +149,38 @@ def _dct_format_s(self) -> str | None:
         """
         return None
 
-    def _dct_issued_s(self) -> str | None:
-        return None
+    def _dct_issued_s(self) -> str:
+        """Field method dct_issued_s
+
+        Because only a single date is allowed, after analysis of publisher dates from tags
+        260 and 265, it was determined that the date embedded in control field 008 is
+        reliable and accurate.
+        """
+        tag_008_value = self.get_single_tag("008").value  # type: ignore[union-attr]
+        return tag_008_value[7:11]
 
     def _dct_identifier_sm(self) -> list[str]:
-        return []
+        identifiers = []
+
+        # get tag 001 value
+        identifiers.append(self.identifier)
+
+        # get other identifiers from tags
+        identifiers.extend(
+            self.get_multiple_tag_subfield_values(
+                [
+                    ("010", "a"),
+                    ("020", "a"),
+                    ("020", "q"),
```

Comment:
Is `020$q` required? Looking at the [examples](https://www.loc.gov/marc/bibliographic/bd020.html) for this subfield, it appears to be extra information that isn't necessarily the identifying value itself. 🤔

Reply:
Good call, really don't need this.  I was cribbing logic from Transmogrifier for Alma MARC.  But quick analysis shows not helpful:

```
+------------------+--------+
|q                 |count(q)|
+------------------+--------+
|null              |0       |
|(pbk.)            |2       |
|(map)             |6       |
|(ring-binder)     |2       |
|(v. 2)            |2       |
|(gzrm)            |2       |
|(flat)            |3       |
|(book)            |11      |
|(v. 1)            |2       |
|(v. 3)            |2       |
|(v. 4)            |2       |
|(folded and cased)|2       |
|(pmf)             |2       |
|(folded)          |1       |
+------------------+--------+
```